### PR TITLE
English version of the BEM Forum project post

### PR DESCRIPTION
**Pull request by rasulksirov**
Ttranslation of the BEM Forum project post for the Hackathon blog

### CODE REVIEW

File: blog/first-bem-forum/first-bem-forum.en.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+
+Here we want to share our impressions of the BEM hackathon and tell you about the results our team was able to achieve.
+
+![](https://img-fotki.yandex.ru/get/15564/44214498.bb/0_9bbca_d03d24dc_XL.jpg)
+
+## The team
+
+A few words about the team members:
+
+* [Kolya Ilchenko]() — the mentor. I have been with Yandex since 2013, working as an interfaces developer (or front end developer, in common parlance). I'm currently developing the bem.info site and BEM forum, I have knowledge of the full BEM stack and four years of experience in web development. My twitter name is [@tavriaforever](https://twitter.com/tavriaforever).
```
Comment (innabelaya) [inline blog/first-bem-forum/first-bem-forum.en.md]: [Kolya Ilchenko](https://bem.info/authors/ilchenko-nikolay/) — ссылка на профайл Коли

File: blog/first-bem-forum/first-bem-forum.en.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+- [Andrey Kuznecov]() — my teammate at Yandex, interfaces developer, mostly on the server side which is developed with node.js. Has experience in server-side development using Java and Ruby. The times when Andrey first started to code already feature in history books. His twitter name is [@kuznetsov48](https://twitter.com/kuznetsov48).
```
Comment (innabelaya) [inline blog/first-bem-forum/first-bem-forum.en.md]: [Andrey Kuznecov](https://en.bem.info/authors/kuznetsov-andrey/)

**Comment by innabelaya** [comment]
Thank you!

**Comment by rasulksirov** [comment]
Thanks - added the missing links to the text.

**Pull request merged**